src/raftstore/store/clocktime.rs

```diff
@@ -0,0 +1,44 @@
+// Copyright 2016 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub use self::inner::now_monotonic_raw_clocktime;
+
+#[cfg(any(target_os = "macos"))]
+mod inner {
+    use time::{self, Timespec};
+
+    pub fn now_monotonic_raw_clocktime() -> Timespec {
+        // TODO Add monotonic raw clock time impl for macos
+        // Currently use the `time::get_time()` instead.
+        time::get_time()
```

Comment:
get_time is real time, not monotonic time. 
you can use `get_precise_ns` and then covert to Timespec instead. 

https://github.com/rust-lang-deprecated/time/blob/master/src/sys.rs#L210